src/main/java/mil/dds/anet/threads/AnetEmailWorker.java

```diff
@@ -94,7 +97,11 @@ protected void runInternal(Instant now, JobHistory jobHistory, Map<String, Objec
         }
         processedEmails.add(email.getId());
       } catch (Throwable t) {
-        logger.error("Error sending email", t);
+        logger.atError().setCause(t).setMessage("Error sending email #{}").addArgument(email::getId)
+            .log();
```

Comment:
Please simplify this to: `logger.error("Error sending email {}", email.getId(), t);`